resources/carrier/en/351.txt

```diff
@@ -13,14 +13,37 @@
 # limitations under the License.
```

Comment:
\# Source: https://www.anacom.pt/pnn/pesquisa.do
\# Results of mobile ranges: https://anacom.pt/pnn/pnnSearch.do?channel=&jscript=on&languageId=0&ssl=false&dataInicioDia=dd&dataInicioMes=mm&dataInicioAno=yyyy&dataFimDia=dd&dataFimMes=mm&dataFimAno=yyyy&assunto=&estado=NON&css=143&css=144&css=145&css=150

Could you please add the above two lines too here.

Reply:
And done!